src/bin/bench.rs

```diff
@@ -139,6 +142,8 @@ pub fn execute(options: Options, config: &mut Config) -> CliResult {
         },
     };
 
+    options.arg_args.extend(options.arg_benchname.into_iter());
```

Comment:
Is it correct to `append` the argument here? I would naturally think that one should `prepend` it instead?

That is, `cargo test foo -- bar` should probably pass `foo bar` to the test harness, and not `bar foo`. However, I don't actually know what is the actual behavior here =)

Reply:
ah, good point. Done.